NineChronicles.Headless.Tests/GraphTypes/ActionQueryTest.cs

```diff
@@ -485,29 +487,64 @@ public async Task PrepareRewardAssets(bool mintersExist, int expectedCount)
-            Assert.Equal(2, action.Recipients.Count);
-            Assert.All(action.Recipients, recipient => Assert.Equal(sender, recipient.recipient));
-            Assert.All(action.Recipients, recipient => Assert.Equal(100, recipient.amount.MajorUnit));
-            Assert.All(action.Recipients, recipient => Assert.Null(recipient.amount.Currency.Minters));
-            foreach (var (ticker, decimalPlaces) in new[] { ("CRYSTAL", 18), ("RUNE_FENRIR1", 0) })
+            if (exc)
             {
-                var recipient = action.Recipients.First(r => r.amount.Currency.Ticker == ticker);
-                Assert.Equal(decimalPlaces, recipient.amount.Currency.DecimalPlaces);
+                var error = Assert.Single(queryResult.Errors!);
+                Assert.Contains($"recipients must be less than or equal {Nekoyume.Action.TransferAssets.RecipientsCapacity}.", error.Message);
             }
+            else
+            {
+                Assert.Null(queryResult.Errors);
+                var data = (Dictionary<string, object>)((ExecutionNode)queryResult.Data!).ToValue()!;
+                var plainValue = _codec.Decode(ByteUtil.ParseHex((string)data["transferAssets"]));
+                Assert.IsType<Dictionary>(plainValue);
+                var polymorphicAction = DeserializeNCAction(plainValue);
+                var action = Assert.IsType<TransferAssets>(polymorphicAction.InnerAction);
+
+                Assert.Equal(sender, action.Sender);
+                Assert.Equal(2, action.Recipients.Count);
+                Assert.All(action.Recipients, recipient => Assert.Equal(sender, recipient.recipient));
+                Assert.All(action.Recipients, recipient => Assert.Equal(100, recipient.amount.MajorUnit));
+                Assert.All(action.Recipients, recipient => Assert.Null(recipient.amount.Currency.Minters));
+                foreach (var (ticker, decimalPlaces) in new[] { ("CRYSTAL", 18), ("RUNE_FENRIR1", 0) })
+                {
+                    var recipient = action.Recipients.First(r => r.amount.Currency.Ticker == ticker);
+                    Assert.Equal(decimalPlaces, recipient.amount.Currency.DecimalPlaces);
+                }
+            }
+        }
+
+        [Fact]
+        public async Task ActivateAccount()
+        {
+            var nonce = new byte[] { 0x00, 0x01, 0x02, 0x03 };
+            var privateKey = new PrivateKey();
+            (ActivationKey activationKey, PendingActivationState _) = ActivationKey.Create(privateKey, nonce);
+            string invitationCode = activationKey.Encode();
+            var query = $"{{ activateAccount(invitationCode: \"{invitationCode}\") }}";
+            var queryResult = await ExecuteQueryAsync<ActionQuery>(query, standaloneContext: _standaloneContext);
+            var data = (Dictionary<string, object>)((ExecutionNode)queryResult.Data!).ToValue()!;
+            var plainValue = _codec.Decode(ByteUtil.ParseHex((string)data["transferAssets"]));
```

Comment:
It seems there is no assertion, is it intended?

Reply:
my mistake included an unintended change.